scripts/polyrith_sage_helper.py

```diff
@@ -15,20 +17,41 @@ def power_to_string(var: int, pow: int) -> str:
         return var_s
     return mk_app("poly.pow", var_s, str(pow))
 
-def sum_to_string(terms: list[str]) -> str:
-    terms = iter(terms)
+@dataclass
+class MonomForm:
+    pos_form: str
+    neg_form: Optional[str] = None
```

Comment:
What exactly do these two fields mean? Can you add a comment?

Reply:
Done!